packages/react/src/primitives/types/buttonGroup.ts

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+
+import { AriaProps } from './base';
+import { BaseStyleProps } from './style';
+import { ButtonProps } from './button';
+import { FlexProps } from './flex';
+
+export interface ButtonGroupProps
+  extends AriaProps,
+    BaseStyleProps,
+    FlexProps,
+    Pick<ButtonProps, 'size' | 'variation'> {
+  children: React.ReactNode;
```

Comment:
is this 100% necessary? Because `React.FC<X>` should add this property.

Reply:
Me and Scott discussed this before. This aims to make the `children` required, which is what we expected, to prevent our users using it like `<ButtonGroup />`. React.FC<X> doesn't force the `children` to be present and thus users won't get any syntax error if they try to do so.

<img width="736" alt="Screen Shot 2021-10-06 at 12 05 24 PM" src="https://user-images.githubusercontent.com/40295569/136266980-107c96df-bd48-4d08-96b0-36f90e987fd8.png">

versus

<img width="525" alt="Screen Shot 2021-10-06 at 12 07 09 PM" src="https://user-images.githubusercontent.com/40295569/136267223-8a8d2d4d-109e-4727-bca3-0f26eb8514d9.png">

Reply:
So I think it makes sense to make it required for group related primitive. Same applies to `ToggleButton`

Reply:
awesome! thanks. I don't have more comments, so I'm approving it.